src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -39,52 +40,25 @@ public void execute_validIndexUnfilteredList_success() {
     }
 
     @Test
-    public void execute_invalidIndexUnfilteredList_throwsCommandException() {
-        Index outOfBoundIndex = Index.fromOneBased(model.getFilteredModuleList().size() + 1);
-        DeleteCommand deleteCommand = new DeleteCommand(outOfBoundIndex);
-
-        assertCommandFailure(deleteCommand, model, Messages.MESSAGE_INVALID_MODULE_DISPLAYED_INDEX);
-    }
-
-    @Test
-    public void execute_validIndexFilteredList_success() {
-        showModuleAtIndex(model, INDEX_FIRST_MODULE);
-
-        Module moduleToDelete = model.getFilteredModuleList().get(INDEX_FIRST_MODULE.getZeroBased());
-        DeleteCommand deleteCommand = new DeleteCommand(INDEX_FIRST_MODULE);
-
-        String expectedMessage = String.format(DeleteCommand.MESSAGE_DELETE_MODULE_SUCCESS, moduleToDelete);
-
-        Model expectedModel = new ModelManager(model.getGradPad(), new UserPrefs());
-        expectedModel.deleteModule(moduleToDelete);
-        showNoModule(expectedModel);
-
-        assertCommandSuccess(deleteCommand, model, expectedMessage, expectedModel);
-    }
-
-    @Test
-    public void execute_invalidIndexFilteredList_throwsCommandException() {
-        showModuleAtIndex(model, INDEX_FIRST_MODULE);
-
-        Index outOfBoundIndex = INDEX_SECOND_MODULE;
-        // ensures that outOfBoundIndex is still in bounds of GradPad list
-        assertTrue(outOfBoundIndex.getZeroBased() < model.getGradPad().getModuleList().size());
-
-        DeleteCommand deleteCommand = new DeleteCommand(outOfBoundIndex);
+    public void execute_invalidModuleCodeUnfilteredList_throwsCommandException() {
+        ModuleCode invalidModuleCode = new ModuleCode("AA0000");
+        // is there a better way to test an invalid module code?
```

Comment:
I don't think so? HAHA
I think there are only 2 equivalence classes for the inputs to `DeleteCommand`:
- Module code exists in the list 
- Module code doesn't exist in the list 

Since you've covered both I think we're good :D